data/mods/More_Makeshift_Items/mb_item_groups.json

```diff
@@ -0,0 +1,46 @@
+[
+  {
+    "type" : "item_group",
+    "id" : "pawn",
+    "items":[
+      ["sword_bayonet", 10],

```

Comment:
I have tested this, they don't override but are added to the item_group. You are thinking monster_groups which do override and need the whole thing written.